po/messages.pot

```diff
@@ -46,45 +46,49 @@ msgid "Idle threshold"
 msgstr ""
 
 #: src/resources/ui/preferences.ui:34
+msgid "Idle animation"
```

Comment:
Wouldn't it be better to call it `"Animated idle"`?

Reply:
I decided on `Idle animation` due to the presence of `Idle threshold`, however I will go ahead and rename everything.